.github/pull_request_template.md

```diff
@@ -9,3 +9,6 @@
 ## Review notes
 
 <!-- Add notable context, discussion items, and anything else that would be helpful for a reviewer to know. -->
+
+## Checklist
```

Comment:
We decided to remove the checklist stuff from the template previously. This is a useful warning though so interested to hear what the rest of the team think. We can always add/remove it later so no need to not merge this.